src/cli/test/run.spec.ts

```diff
@@ -0,0 +1,154 @@
+import type * as d from '../../declarations';
+import * as coreCompiler from '@stencil/core/compiler';
+import { mockCompilerSystem, mockConfig, mockLogger as createMockLogger } from '@stencil/core/testing';
+import * as ParseFlags from '../parse-flags';
+import { run, runTask } from '../run';
+import * as HelpTask from '../task-help';
+import * as TelemetryTask from '../task-telemetry';
+import { createTestingSystem } from '../../testing/testing-sys';
+
+describe('run', () => {
+  describe('run()', () => {
+    let cliInitOptions: d.CliInitOptions;
+    let mockLogger: d.Logger;
+    let mockSystem: d.CompilerSystem;
+
+    let parseFlagsSpy: jest.SpyInstance<
+      ReturnType<typeof ParseFlags.parseFlags>,
+      Parameters<typeof ParseFlags.parseFlags>
+    >;
+
+    beforeEach(() => {
+      mockLogger = createMockLogger();
+      mockSystem = createTestingSystem();
+
+      cliInitOptions = {
+        args: [],
+        logger: mockLogger,
+        sys: mockSystem,
+      };
+
+      parseFlagsSpy = jest.spyOn(ParseFlags, 'parseFlags');
+      parseFlagsSpy.mockReturnValue({
+        task: 'help',
+      });
```

Comment:
should this `.mockReturnValue` bit here be nested under the `'help task'` describe block, below? j/w because it seems specific to testing the 'help' task, although I guess we could also leave it here as a default value for future tests of other tasks

Reply:
> although I guess we could also leave it here as a default value for future tests of other tasks

that was my thought, since 'help' returns early, it seemed like a reasonable/safe default for other tests.  I'll add a comment here explaining that

Reply:
👍 dc3055292